Procfile.dev

```diff
@@ -0,0 +1,3 @@
+tunnel: script/tunnel
+web: nodemon --exec "npm start"
+worker: nodemon --exec "npm run worker"
```

Comment:
Just a note that I had to `npm i -g nodemon` in order for this to work, which I didn't need before due to how `npm run test` resolve binaries. May want to include it in `script/bootstrap` or you could just do `./node_modules/.bin/nodemon`.

Reply:
Good find. I've tweaked it to use commands from `package.json` so it should just work now.